src/services/tracking/useTracking.js

```diff
@@ -46,5 +48,21 @@ export function useTracking() {
   useTrackFeatureFlags(user) // TODO: Can probably delete
   useUpdatePendoWithOwner(user)
 
+  let sentryUser = null
+  if (user?.email) {
+    sentryUser = {
+      email: user.email,
+    }
+  }
+
+  if (user?.user?.username) {
+    sentryUser = {
+      ...sentryUser,
+      username: user.user.username,
+    }
+  }
```

Comment:
```suggestion
  const maybeSentryUser = {}
  if (user?.email) {
    maybeSentryUser.email = user.email
  }
  if (user?.user?.username) {
    maybeSentryUser.username: user.user.username
  }
  
  const sentryUser = Object.keys(maybeSentryUser).length === 0 ? null : maybeSentryUser
  Sentry.setUser(sentryUser)
```

Reply:
Personally, I think the former was more readable